docs/blueprints/EP036.rst

```diff
@@ -130,16 +173,35 @@ This event should sent before any NApp sends its first flow to be published.
     "coloring": {"base": 0}
   }
 
-Deploying pipeline
+
+
+Endpoint API
+============
+  
+  - GET `/api/kytos/multi_table/v2/` - Get all pipelines in the database. Parameters enabled and disabled are allowed.
+
+  - POST `/api/kytos/multi_table/v2/<pipeline_id>/enable` - Enable pipeline to be applied to all switches

```

Comment:
Can you also include a resource "`pipeline`" in the scope of the routes after the version? like `v1/pipeline/...`, same way that we've been standardizing and maintaining for kytos-ng org NApps? 

We probably won't have different resources other than a pipeline, but if we ever do, then having this `pipeline` scope allow us to easily evolve this API on day.